examples/train/dspark_qwen3_4b_gsm8k_online.sh

```
@@ -0,0 +1,141 @@
+    --vllm-endpoint "http://127.0.0.1:${VLLM_PORT}/v1" \
+    --save-path "$OUTPUT_DIR/checkpoints" \
+    --draft-vocab-size "$DRAFT_VOCAB_SIZE" \
+    --epochs "$EPOCHS" \
+    --lr "$LR" \
+    --total-seq-len "$SEQ_LENGTH" \
+    --speculator-type "$SPECULATOR_TYPE" \
+    --block-size "$BLOCK_SIZE" \
+    --num-anchors "$NUM_ANCHORS" \
+    --num-layers "$NUM_LAYERS" \
+    --draft-attn-impl eager 
+    --target-layer-ids $TARGET_LAYER_IDS \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)